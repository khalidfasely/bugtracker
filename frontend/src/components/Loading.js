import photo from '../images/loader.gif';

const Loading = () => (
    <div className="loading-container">
        <img src={photo} alt='loading GIF' className="loading-img" />
    </div>
);

export default Loading;