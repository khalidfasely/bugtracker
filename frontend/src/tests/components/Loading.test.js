import React from "react";
import { render } from "@testing-library/react";

import Loading from "../../components/Loading";

test('Should render Loading component correctly', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
});