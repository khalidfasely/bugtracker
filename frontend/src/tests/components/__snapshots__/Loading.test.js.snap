// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Loading component correctly 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
